docs/specs/iota_did_method_spec.md

```diff
@@ -0,0 +1,324 @@
+
+### IOTA-Tag
+
+The IOTA tag references an indexation which resolves to the initial DID Messages. 
+
+#### Generation
+
+The following steps MUST be taken to generate a valid Tag:
+* Generate an asymmetric keypair using a supported verification method type.
+* Hash the public key using `BLAKE2b-256` and encode using base58.
+
+Note that this public key MUST also be embedded into the DID Document (See [CRUD: Create](#Create)).
+
+## DID Messages
+
+DID Documents associated with the `did:iota` method consist of a chain of data messages, called "DID messages", published to a Tangle. The Tangle has no understanding of DID messages and acts purely as an immutable database. The chain of DID messages and the resulting DID Document must therefore be validated on the client side. 
+
+A DID message can be part of one of two different message chains, the "Integration Chain" (Int Chain) and the "Differentiation Chain" (Diff Chain). The Integration Chain is a chain of "DID Integration Messages" that contain JSON formatted DID Documents as per the W3C standard for DID. The Diff Chain is a chain of "DID Diff Messages" that contain JSON objects which only list the differences between the previous DID Document and the next state. 
+
+### Previous Message Id
+
+Any DID message uploaded to the Tangle, with the exception of the very first DID message that creates the DID, MUST contain a `previous_message_id` field. This field MUST carry the MessageId, an IOTA indexation for a single message, of the previous DID Document that is updated with this DID message. This value SHOULD be used to order DID messages during the resolving procedure. If two or more DID messages reference the same `previous_message_id` an ordering conflict is identified and is resolved using a deterministic ordering mechanism. 
+
+Example of an IOTA MessageId:
+```json
+"previous_message_id": "cd8bb7baca6bbfa1de7813bd1753a2de026b6ec75dba8a3cf32c0d4cf6038917"
+```
+
+### Signing Key
+
+DID Documents published to the Tangle must be cryptographically signed. As such the DID Document MUST include one verification method with a public key. It is recommended, for security reasons, to not use this keypair for other purposes as the control over this private key is vital for controlling the identity. It is RECOMMENDED to name this public key #_sign-x, where x is the index of the signing key, which is incredemented everytime the signing key is updated, starting at index 1. 
+
+### Autonomy of DID Integration Messages
+
+A DID Integration message MUST contain a valid DID Document according to the W3C DID standard. In addition, the message has further restrictions:
+
+* The DID Document MUST always contain one or more verification methods with a public key in the `verificationMethod` object of the DID Document. It is RECOMMENDED to only use this key for updating the DID Document and name this public key #_sign-x. 
```

Comment:
Nit: `verificationMethod` _property_ of the DID Document.

Looks like the spec uses the term "property": https://www.w3.org/TR/did-core/#did-document-properties